src/meepgeom.cpp

```diff
@@ -2567,4 +2567,26 @@ void material_grids_addgradient(meep::realnum *v, size_t ng, std::complex<double
   }
 }
 
+void get_epsilon_grid(geometric_object_list gobj_list,
+                      material_type_list mlist,
+                      material_type _default_material,
+                      bool _ensure_periodicity,
+                      meep::grid_volume gv,
+                      vector3 cell_size,
+                      vector3 cell_center,
+                      int nx, double *x,
+                      int ny, double *y,
+                      int nz, double *z,
+                      double *grid_vals) {
+  init_libctl(_default_material, _ensure_periodicity, &gv,
```

Comment:
seems like you need to have `dim =gv.dim` before calling `geps`?

Reply:
I added this even though it doesn't not seem to affect the results (i.e., the results are the same with or without this line).